### DISCUSSION TITLE
List of websites that using docfx

### DISCUSSION THREAD

**Discussion by filzrev** [dotnet/docfx#9002] [General]
I've curated site lists that use docfx.
If you know of any other interesting sites, please let me know. I’ll add them to the list.

| Docs Site                              | Source Repository                                                | GitHub Stars                                                                                             |
|----------------------------------------|------------------------------------------------------------------|----------------------------------------------------------------------------------------------------------|
| [Benchmark.NET][01]                    | https://github.com/dotnet/BenchmarkDotNet                        | ![GitHub Stars](https://img.shields.io/github/stars/dotnet/BenchmarkDotNet.svg?style=social)             |
| [DNN Docs][02]                         | https://github.com/DNNCommunity/DNNDocs                          | ![GitHub Stars](https://img.shields.io/github/stars/DNNCommunity/DNNDocs.svg?style=social)               |
| [Uno Platform][03]                     | https://github.com/unoplatform/uno                               | ![GitHub Stars](https://img.shields.io/github/stars/unoplatform/uno.svg?style=social)                    |
| [SuperOffice Platform][04]             | https://github.com/SuperOfficeDocs/superoffice-docs              | ![GitHub Stars](https://img.shields.io/github/stars/SuperOfficeDocs/superoffice-docs.svg?style=social)   |
| [IgniteUI for Web Components][05]      | https://github.com/IgniteUI/igniteui-xplat-docs                  | ![GitHub Stars](https://img.shields.io/github/stars/IgniteUI/igniteui-docfx.svg?style=social)            |
|                                        | https://github.com/IgniteUI/igniteui-docfx                       | ![GitHub Stars](https://img.shields.io/github/stars/IgniteUI/igniteui-docfx.svg?style=social)            |
|                                        | https://github.com/IgniteUI/igniteui-docfx-template              | ![GitHub Stars](https://img.shields.io/github/stars/IgniteUI/igniteui-docfx-template.svg?style=social)   |
| [Stride documentation][06]             | https://github.com/stride3d/stride-docs                          | ![GitHub Stars](https://img.shields.io/github/stars/stride3d/stride-docs.svg?style=social)               |
| [Flax Engine Documentation][07]        | https://github.com/FlaxEngine/FlaxDocs                           | ![GitHub Stars](https://img.shields.io/github/stars/FlaxEngine/FlaxDocs.svg?style=social)                |
| [dotnet-config][08]                    | https://github.com/dotnetconfig/dotnet-config                    | ![GitHub Stars](https://img.shields.io/github/stars/dotnetconfig/dotnet-config.svg?style=social)         |
| [Sorted Docs][09]                      | `N/A`                                                            |                                                                                                          |
| [Customer Canvas][10]                  | `N/A`                                                            |                                                                                                          |
| [esfx][11]                             | https://github.com/esfx/esfx                                     | ![GitHub Stars](https://img.shields.io/github/stars/esfx/esfx.svg?style=social)                          |
| [Google APIs][12]                      | https://github.com/googleapis/doc-pipeline                       | ![GitHub Stars](https://img.shields.io/github/stars/googleapis/doc-pipeline.svg?style=social)            |
|                                        | https://github.com/googleapis/google-cloud-dotnet                | ![GitHub Stars](https://img.shields.io/github/stars/googleapis/google-cloud-dotnet.svg?style=social)     |
| [DocuWare Developer Documentation][13] | `N/A`                                                            |                                                                                                          |
| [JsonApiDotNetCore][14]                | https://github.com/json-api-dotnet/JsonApiDotNetCore             | ![GitHub Stars](https://img.shields.io/github/stars/json-api-dotnet/JsonApiDotNetCore.svg?style=social)  |
| [WPF UI][15]                           | https://github.com/lepoco/wpfui                                  | ![GitHub Stars](https://img.shields.io/github/stars/lepoco/wpfui.svg?style=social)                       |
| [Akka .NET][16]                        | https://github.com/akkadotnet/akka.net                           | ![GitHub Stars](https://img.shields.io/github/stars/akkadotnet/akka.net.svg?style=social)                |
| [Cratis][17]                           | https://github.com/aksio-insurtech/Cratis                        | ![GitHub Stars](https://img.shields.io/github/stars/aksio-insurtech/Cratis.svg?style=social)             |
| [Polly Docs][18]                       | https://github.com/App-vNext/Polly                               | ![GitHub Stars](https://img.shields.io/github/stars/App-vNext/Polly.svg?style=social)                    |
| [YARP][19]                             | https://github.com/microsoft/reverse-proxy                       | ![GitHub Stars](https://img.shields.io/github/stars/microsoft/reverse-proxy.svg?style=social)            |
| [LitMotion][20]                        | https://github.com/AnnulusGames/LitMotion                        | ![GitHub Stars](https://img.shields.io/github/stars/AnnulusGames/LitMotion.svg?style=social)             |
| [Npgsql][21]                           | https://github.com/npgsql/doc                                    | ![GitHub Stars](https://img.shields.io/github/stars/npgsql/doc.svg?style=social)                         |
| [NUnit Documentation Site][22]         | https://github.com/nunit/docs                                    | ![GitHub Stars](https://img.shields.io/github/stars/nunit/docs.svg?style=social)                         |
| [nopCommerce Documentation][23]        | https://github.com/nopSolutions/nopCommerce-Docs                 | ![GitHub Stars](https://img.shields.io/github/stars/nopSolutions/nopCommerce-Docs.svg?style=social)      |
| [StockSharp][24]                       | https://github.com/StockSharp/doc                                | ![GitHub Stars](https://img.shields.io/github/stars/StockSharp/doc.svg?style=social)                     |
| [Discord.Net][25]                      | https://github.com/discord-net/Discord.Net                       | ![GitHub Stars](https://img.shields.io/github/stars/discord-net/Discord.Net.svg?style=social)            |
| [LINQ to DB][26]                       | https://github.com/linq2db/linq2db                               | ![GitHub Stars](https://img.shields.io/github/stars/linq2db/linq2db.svg?style=social)                    |
| [NRules][27]                           | https://github.com/NRules/NRules                                 | ![GitHub Stars](https://img.shields.io/github/stars/NRules/NRules.svg?style=social)                      |
| [MongoDB .NET/C# Driver API Reference][28]                | https://github.com/mongodb/mongo-csharp-driver| ![GitHub Stars](https://img.shields.io/github/stars/mongodb/mongo-csharp-driver.svg?style=social)        |
| [idunno.Bluesky][29]                   | https://github.com/blowdart/idunno.Bluesky                       | ![GitHub Stars](https://img.shields.io/github/stars/blowdart/idunno.Bluesky.svg?style=social)            |
| [api.xunit.net][30]                    | https://github.com/xunit/api.xunit.net                           | ![GitHub Stars](https://img.shields.io/github/stars/xunit/api.xunit.net.svg?style=social)                |
| [new.xunit.net][31]                    | https://github.com/xunit/xunit.net                               | ![GitHub Stars](https://img.shields.io/github/stars/xunit/xunit.net.svg?style=social)                    |

> [!NOTE]
> GitHub star badges are displayed based on the starts of the `docs' repository.

[01]: https://benchmarkdotnet.org
[02]: https://docs.dnncommunity.org
[03]: https://platform.uno/docs/articles/intro.html
[04]: http://docs.superoffice.com
[05]: https://www.infragistics.com/products/ignite-ui-web-components/web-components/components/general-getting-started
[06]: https://doc.stride3d.net/latest/en
[07]: https://docs.flaxengine.com
[08]: https://dotnetconfig.org
[09]: https://docs-preview.sorted.com
[10]: https://customerscanvas.com/dev/index.html
[11]: https://esfx.js.org/esfx
[12]: https://cloud.google.com/dotnet/docs/reference
[13]: https://developer.docuware.com
[14]: https://www.jsonapi.net/getting-started/install.html
[15]: https://wpfui.lepo.co
[16]: https://getakka.net
[17]: https://www.cratis.io
[18]: https://www.pollydocs.org
[19]: https://microsoft.github.io/reverse-proxy
[20]: https://annulusgames.github.io/LitMotion
[21]: https://www.npgsql.org
[22]: https://docs.nunit.org
[23]: https://docs.nopcommerce.com
[24]: https://doc.stocksharp.com
[25]: https://docs.discordnet.dev
[26]: https://linq2db.github.io
[27]: https://nrules.net
[28]: https://mongodb.github.io/mongo-csharp-driver/2.25.0/api/index.html
[29]: https://bluesky.idunno.dev
[30]: https://api.xunit.net
[31]: https://xunit.net

**Comment by yufeih** [comment]
Sounds like a good idea to put this list in https://dotnet.github.io/docfx

**Comment by filzrev** [comment]
Above site list is intended for samples for customize templates.

For official documentation.
I though a system like [Docusaurus Site Showcase](https://docusaurus.io/showcase) would be desirable.
(List up only when site owner approved)

**Comment by mikasoukhov** [comment]
We use it https://doc.stocksharp.com/ and https://github.com/stockSharp/StockSharp

**Comment by ogulcanturan** [comment]
https://docs.opensettings.net/ is built with DocFX. Repo: https://github.com/OpenSettings/open-settings-docs

**Comment by lcarrere** [comment]
LM-Kit.NET https://docs.lm-kit.com/lm-kit-net/index.html

**Comment by jo3w4rd** [comment]
Most Unity package documentation is generated with DocFX. Here's a partial list: https://docs.unity3d.com/Manual/pack-safe.html. (The core Unity manual and API reference use an older, in-house publishing system.)
